app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -72,7 +72,13 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
 
     CONSUMER_RPM=$(mktemp --suffix .rpm)
     curl --silent --output $CONSUMER_RPM <%= subscription_manager_configuration_url(hostname: @url_host) %>
-    yum localinstall $CONSUMER_RPM -y
+
+    if [ "$(cat /proc/sys/crypto/fips_enabled)" = "1" ]; then
+        rpm -ivh --nodigest --nofiledigest $CONSUMER_RPM
```

Comment:
This deserves a comment because it's a workaround. Normally we open an issue for that and link it in the code. It probably means we should start generating a consumer RPM with a secure digest instead of simply not checking it.

That also makes me doubt if we even need this in the first place or rather we should fix the source.